x-pack/plugins/monitoring/dev_docs/reference/data_collection_modes.md

```diff
@@ -0,0 +1,199 @@
+
+- [Elasticsearch](https://github.com/elastic/beats/tree/main/metricbeat/module/elasticsearch)
+- [Kibana](https://github.com/elastic/beats/tree/main/metricbeat/module/kibana)
+- [Logstash](https://github.com/elastic/beats/tree/main/metricbeat/module/logstash)
+- [Beats](https://github.com/elastic/beats/tree/main/metricbeat/module/beat)
+- [Enterprise Search](https://github.com/elastic/beats/tree/main/x-pack/metricbeat/module/enterprisesearch)
+
+Here's an example using Elasticsearch. All other components implement similar HTTP endpoints and metricbeat modules.
+
+```mermaid
+graph LR
+subgraph Production Elasticsearch
+  ClusterStats["/_cluster/stats"]
+  NodeStats["/_nodes/_local/stats"]
+end
+
+subgraph Monitoring Deployment
+  MonElasticsearch[Elasticsearch]
+end
+
+subgraph Metricbeat
+  ElasticsearchModule[Elasticsearch Module]
+  click ElasticsearchModule "https://github.com/elastic/beats/tree/main/metricbeat/module/elasticsearch"
+end
+
+ElasticsearchModule-.->|poll|ClusterStats
+ElasticsearchModule-.->|poll|NodeStats
+ElasticsearchModule-->|_bulk|MonElasticsearch
+```
+
+You can monitor many components from a single metricbeat process as is typically done during local development. You can also have a dedicated metricbeat process for each instance in a deployment as is done on ESS.
+
+## Logs collection (Filebeat)
+
+Regardless of the metrics collection mode, logs should always be collected using filebeat. Filebeat ships with a module for some stack components that can be used to collect the logs for that component.
+
+- [Elasticsearch](https://github.com/elastic/beats/tree/main/filebeat/module/elasticsearch)
+- [Kibana](https://github.com/elastic/beats/tree/main/filebeat/module/kibana)
+- [Logstash](https://github.com/elastic/beats/tree/main/filebeat/module/logstash)
+ 
+Stack Monitoring will read those same logs, as configured by the `monitoring.ui.logs.index` setting.
+
+For example, in the case of the elasticsearch filebeat module reading slow query logs and writing to the corresponding pipeline on Elasticsearch:
+
+```mermaid
+graph LR
+subgraph Production Elasticsearch
+  SlowQueryLogs[*_index_search_slowlog.log]
+end
+
+subgraph Monitoring Deployment
+  subgraph Elasticsearch
+    Pipeline>filebeat-X.Y.Z-elasticsearch-slowlog-pipeline]
+  end
+end
+
+subgraph Filebeat
+  ElasticsearchModule[Elasticsearch Module]
+end
+
+Disk[(Disk)]
+SlowQueryLogs-.->|write|Disk
+
+ElasticsearchModule-.->|read|Disk
+ElasticsearchModule-->|_bulk|Pipeline
+```
+
+Beats and Enterprise Search don't have filebeat modules, but the logs can be ingested using basic JSON filebeat configurations.
+
+See (internal) https://github.com/elastic/enterprise-search-team/issues/1532 for updates on Enterprise Search filebeat modules and configurations.
+
+## Unified collection
+
+### Package-driven collection
+
+When using package-driven collection, each component in your Elastic stack is given a corresponding fleet package (also known as "integration").
```

Comment:
That's actually what I had, but @jasonrhodes seemed to want package in https://github.com/elastic/kibana/pull/127567#discussion_r826370023

Glad I'm not alone in the confusion 😆

Reply:
The entire company shares this confusion with you :)

All I care about here is that we can't use both terms here as if they are different, when they're (almost) identical (for our purposes here). We could just use integration, but I think calling out the name confusion explicitly like you've got here with the "also known as" is likely better. We can probably drop "fleet" from that sentence, if we want to avoid that specific confusion.

Reply:
Cool. I'll keep it as-is then 👍🏻